pipelinewise/fastsync/commons/tap_postgres.py

```diff
@@ -1,12 +1,15 @@
+from dataclasses import dataclass
 import datetime
 import decimal
 import logging
 import re
 import sys
+from time import time
```

Comment:
nit: not sure if it's Pipelinewise's style to import whole modules rather than functions, though from the other imports it looks like it might be. Certainly it's our internal style to do that, so it's probably preferable to stick to that where it's consistent with the existing code?

Reply:
Yup, good spot. This was just an auto VSCode import I missed!